recipe/meta.yaml

```diff
@@ -2,52 +2,75 @@
 {% set version = "2.10.4" %}
 
 package:
-  name: {{ name }}
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/pydantic_extra_types-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_')}}-{{ version }}.tar.gz
   sha256: bf8236a63d061eb3ecb1b2afa78ba0f97e3f67aa11dbbff56ec90491e8772edc
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: True  # [py<38]
 
 requirements:
   host:
-    - python {{ python_min }}
+    - python
     - hatchling
     - pip
   run:
-    - python >={{ python_min }}
+    - python
     - pydantic >=2.5.2
+    - typing_extensions
   run_constrained:
-    - phonenumbers >=8,<9
-    - pycountry >=23
-    - semver >=3.0.2,<4
-    - python-ulid >=1,<3
     - pendulum >=3.0.0,<4.0.0
+    - phonenumbers >=8,<10
+    - pycountry >=23
+    - python-ulid >=1,<2  # [py<39]
+    - python-ulid >=1,<4  # [py>=39]
+    - semver >=3.0.2 
+    - pymongo >=4.0.0,<5.0.0
     - pytz >=2024.1
-    - tzdata >=2024a
+    - semver >=3.0.2,<3.1.dev0
```

Comment:
There is a duplication of `semver`. I think semver >=3.0.2,<3.1.0 is fine but semver >=3.0.2 is not a problem too.

Reply:
And below in test.requires